apps/src/templates/Certificate.jsx

```diff
@@ -159,6 +172,17 @@ class Certificate extends Component {
             userAge={userAge}
           />
         </div>
+        {showKoreanMinecraftLink && (
+          <div style={styles.extraLinkContainer}>
+            <a
+              href="http://www.mscodingparty.com/certificate.html"
+              target="_blank"
+              style={styles.extraLink}
+            >
+              온라인 코딩 파티 인증서 받으러 가기! (과학기술정보통신부 인증)
+            </a>
+          </div>
+        )}
```

Comment:
I wonder if we want to pass bit of code in as a Child, instead of having the logic live here?  Then this component remains simple, and can render any Child below the certificate, if present.

Reply:
I'll commit an alternate approach in which we just pass this in as a child property.

Reply:
Looks great!